aslam_offline_calibration/kalibr/python/kalibr_common/ConfigReader.py

```diff
@@ -365,8 +471,8 @@ def getAccelerometerStatistics(self):
     
     def setAccelerometerStatistics(self, noise_density, random_walk):
         self.checkAccelerometerStatistics(noise_density, random_walk)
-        self.data["accelerometer_noise_density"] = accelerometer_noise_density
-        self.data["accelerometer_random_walk"] = accelerometer_random_walk
+        self.data["accelerometer_noise_density"] = noise_density
```

Comment:
Was this a bug previously?

Reply:
Looks like it, yes. But it seems it's not used by anyone :-).